### PR TITLE
go-sct: 20180605-eb1e851 -> unstable-2022-01-32

### PR DESCRIPTION
**Pull request by aaronjheng**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/X11/go-sct/default.nix [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+  vendorSha256 = "sha256-Rx5/oORink2QtRcD+JqbyFroWYhuYmuYDzZ391R4Jsw=";
+
+  buildInputs = [ xorg.libX11 xorg.libXrandr wayland.dev ];
 
-  buildInputs = [ xorg.libX11 xorg.libXrandr ];
+  ldflags = [ "-s" "-w" ];
 
   meta = with lib; {
     description = "Color temperature setting library and CLI that operates in a similar way to f.lux and Redshift";
+    homepage = "https://github.com/d4l3k/go-sct";
```
Comment (ckiee) [inline pkgs/tools/X11/go-sct/default.nix:27]:
```suggestion
    homepage = "https://github.com/d4l3k/go-sct";
    mainProgram = "sct";
```

File: pkgs/tools/X11/go-sct/default.nix [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
     license = licenses.mit;
     maintainers = with maintainers; [ cstrahan ];
     platforms = platforms.linux ++ platforms.windows;
```
Comment (ckiee) [inline pkgs/tools/X11/go-sct/default.nix:30]: Not with our go:
```
ckie@cookiemonster ~ -> nix repl
Welcome to Nix 2.9.0pre20220428_4bb111c. Type :? for help.
nix-repl> :l <nixpkgs>
Added 16328 variables.
nix-repl> go.meta.platforms
[ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "i686-linux" "m68k-linux" "mipsel-linux" "mips64el-linux" "powerpc64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux" "s390x-linux" "x86_64-linux" "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin" ]
```
```suggestion
    platforms = go.meta.platforms;
```

**Review by ckiee — CHANGES_REQUESTED** [review]

File: pkgs/tools/X11/go-sct/default.nix (hunk 2 above)
Comment (ckiee) [reply pkgs/tools/X11/go-sct/default.nix:30]: Oh. Neat.

Comment (SuperSandro2000) [inline pkgs/tools/X11/go-sct/default.nix:30]:
```suggestion
```
Go does not work on windows.

**Pull request merged**